ui/app/common/bifrost/bifrostConfig.js

```diff
@@ -26,28 +26,41 @@
   module.provider('bifrostConfig', function () {
 
     let bifrostUrl;
+    let bifrostWsUrl;
 
     function setBifrostUrl(url) {
       bifrostUrl = url;
     }
 
+    function setBifrostWsUrl(url) {
+      bifrostWsUrl = url;
+    }
+
     function getBifrostUrl() {
       return bifrostUrl;
     }
 
+    function getBifrostWsUrl() {
+      return bifrostWsUrl;
+    }
+
+
     function getBifrostHost() {
```

Comment:
If this method uses `bifrostWsUrl` now, then it should be called `getBifrostWsHost` (see other comments).